spec/cms/controllers/training/modules_controller_spec.rb

```diff
@@ -6,7 +6,7 @@
     before do
       skip 'WIP' unless Rails.application.cms?
 
-      sign_in create(:user, :registered)
+      sign_in create(:user, :registered, :email_opt_in)
```

Comment:
Should change the trait as part of the renaming.

Reply:
Renamed [088d9a3](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/659/commits/088d9a3236a1d392aaef71939822b94962408d63)

Reply:
I'd have not suggested it if there were an equivalent `email_opt_out` trait. Do you think one is needed?

Reply:
Think it could probably be useful particularly when testing some of the future tickets around the email automation. Will add one

Reply:
[20215f3](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/659/commits/20215f3d038223d33d245c39505f1ec020355841)